examples/collection-only/collection.json

```
@@ -44,13 +44,14 @@
     }
   },
   "assets": {
-    "thumbnail": {
-      "href": "thumbnail.png",
-      "title": "Preview",
-      "type": "image/png",
+    "metadata_iso_19139": {
       "roles": [
-        "thumbnail"
-      ]
+        "metadata",
+        "iso-19139"
+      ],
+      "href": "https://storage.googleapis.com/open-cogs/stac-examples/sentinel-2-iso-19139.xml",
+      "title": "ISO 19139 metadata",
+      "type": "vnd.iso.19139+xml"
```

That doesn't seem like a valid media type.

---

Yeah... I got it from spacebel, and found reference in [geoserver](https://geoserver.geo-solutions.it/downloads/releases/2.8.x-ld/doc/extensions/inspire/using.html) and [mapproxy](https://mapproxy.org/docs/latest/inspire.html) so seemed like it may be some official INSPIRE thing? 

I'm also fine to just use xml type...

---

Some googling brought up that it should probably be:

```suggestion
      "type": "application/vnd.iso.19139+xml"
```